src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/judgeapproval/ApproveOrderService.java

```diff
@@ -16,6 +19,28 @@ public class ApproveOrderService {
 
     private final JudgeApprovalResolver judgeApprovalResolver;
 
+    private static final String ORDERS_APPROVED_CONFIRMATION_BODY_FORMAT =
+        "<br>The draft orders (%s) for the case have been approved. "
+            + "They are now ready for caseworker review.";
+
+    private static final String ORDERS_REQUIRE_REPRESENTATIVE_CHANGE_CONFIRMATION_BODY_FORMAT =
+        "<br>You have said that the legal representative needs to make some changes to the draft order(s). "
+            + "They have been sent an email with your reasons for rejecting the orders (%s)";
+
+    private static final String ORDERS_AMENDED_CHANGE_CONFIRMATION_BODY_FORMAT =
+        "<br>You have amended the draft orders (%s). They are now ready for caseworker review";
+
+    private static final String ORDERS_REVIEW_LATER_CONFIRMATION_BODY_FORMAT =
+        "<br>You have said you will review draft orders (%s) later. These will remain on the "
+            + "['Draft Orders' tab](/cases/case-details/%s#Draft%%20orders).";
+
+
+    private final List<String> ordersApproved = new ArrayList<>();
+    private final List<String> ordersRepresentativeChanges = new ArrayList<>();
+    private final List<String> ordersChanged = new ArrayList<>();
+    private final List<String> ordersReviewLater = new ArrayList<>();
```

Comment:
should they be declared withiin `buildConfirmationBody`?

Reply:
I suspect that if you run the second approve order event, the file names will keep accumulating here.